nncf/openvino/quantization/quantize_model.py

```diff
@@ -91,6 +93,57 @@ def dump_parameters(model: ov.Model, parameters: Dict, path: Optional[List] = No
         nncf_logger.debug(f"Unable to dump optimization parameters due to error: {e}")
 
 
+@tracked_function(NNCF_OV_CATEGORY, [CompressionStartedWithQuantizeApi(), "target_device", "preset"])
+def native_quantize_if_op_impl(
+    model: ov.Model,
+    calibration_dataset: Dataset,
+    preset: QuantizationPreset = QuantizationPreset.PERFORMANCE,
+    target_device: TargetDevice = TargetDevice.ANY,
+    subset_size: int = 300,
+    fast_bias_correction: bool = True,
+    model_type: Optional[ModelType] = None,
+    ignored_scope: Optional[IgnoredScope] = None,
+    advanced_parameters: Optional[AdvancedQuantizationParameters] = None,
+) -> ov.Model:
+    """
+    Implementation of the `quantize()` method for the OpenVINO backend via the OpenVINO Runtime API.
+    """
+    if not fast_bias_correction:
+        raise NotImplementedError(
+            "The BiasCorrection algorithm is not supported for OpenVINO models with If operation."
+        )
```

Comment:
Does the `FastBiasCorrection` algorithm work for the `ifmodel`? If not, then another condition should be used.
```python
if not advanced_parameters.disable_bias_correction:
    ...
```

Do the SmoothQuant and ChannelAlignment algorithms work for `ifmodel`?